crates/astria-sequencer/src/mempool.rs

```diff
@@ -117,6 +133,81 @@ impl std::hash::Hash for EnqueuedTransaction {
     }
 }
 
+#[derive(Debug)]
+pub(crate) struct TransactionExpired;
+
+impl std::fmt::Display for TransactionExpired {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "Transaction expired")
+    }
+}
+
+const TX_TTL: Duration = Duration::from_secs(600); // 10 minutes 
+const REMOVAL_CACHE_SIZE: NonZeroUsize = unsafe { NonZeroUsize::new_unchecked(4096) }; // TODO make configuration variable 
+const TX_REMOVAL_TTL: Duration = Duration::from_secs(30); // 30 seconds
+
+/// `RemovalCache` is a cache used for signaling to `CometBFT` when a
+/// transaction can be removed from the `CometBFT` mempool outside of the
+/// `CheckTx` checks.
```

Comment:
this is checked in `check_tx` right? there isn't a way to tell cometbft to remove something from the mempool without cometbft calling `check_tx` on it

Reply:
My b, yeah the comment should read 'other CheckTX checks'